### ISSUE TITLE
[SUGGESTION] Cushion Bricks That Can Be Sat On

### ISSUE THREAD

**Issue by Senakysam** [tsunamayo/Skywanderers#1080]
I love the new seats, they will look pretty good along with the new textures and shiny effects when building my ships. But, when looking at them, I have a hard time seeing how I could use them as a base for custom seats without leaving gaps. So for this suggestion, I'd like to see cushions that the player can enter/sit on so that players have the option of 100% custom brick seats.

**Comment by gmunga5** [comment]
I also like the look of them but don't see anyway to add a button on the armrest as before. Do you have plans to make that possible in some way? Would be nice for something like a spinning chair.

**Comment by tsunamayo** [comment]
Yes for sure it is less customizable, you can put a button but it will look a bit odd. So I was thinking of adding a very basic seat (or maybe a cushion, I need to think more about that) on top of that for people who want to make their own. Also the couch design sucks, I think I will redo it when I have time. I will have a build for you guy soon, I am still doing some big refactor and need to work on a few more blocks. I hope you will like it...

**Comment by tsunamayo** [comment]
I still need to work on that...
